prowler/providers/aws/services/elbv2/elbv2_service.py

```diff
@@ -51,6 +54,57 @@ def __describe_load_balancers__(self, regional_client):
                 f"{regional_client.region} -- {error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
             )
 
+    def __describe_target_groups__(self, regional_client):
+        logger.info("ELBv2 - Describing target groups...")
+        try:
+            for lb in self.loadbalancersv2:
+                try:
+                    if (
+                        lb.scheme == "internet-facing"
+                        and lb.type == "application"
+                        and len(lb.security_groups) > 0
+                    ):
```

Comment:
I'd store all with a check of `public` if this condition is met.

Reply:
Done